index.html

```diff
@@ -0,0 +1,246 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+
+    <link rel="stylesheet" href="style.css">
+    <title>TechLife Development Company</title>
+</head>
+<body>
+<!-- Nav banner -->
+<div class="d-sm-none d-none d-md-block upper-nav pe-2">
+    <ul class="nav justify-content-end py-1 nav-banner">
+        <li class="nav-item">
+        <a class="nav-link mt-1" href="#"><i class="bi bi-facebook"></i></a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link mt-1" href="#"><i class="bi bi-twitter"></i></a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link" href="#">English</a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link" href="#">My Page</a>
+        </li>
+        <li class="nav-item">
+        <a class="nav-link" href="#">Logout</a>
+        </li>
+    </ul>
+</div>
```

Comment:
- [ ] Good job so far but I think you should adjust your top bar right position. Please make sure to make it similar to the original design as mentioned by the previous reviewer 👍 

#### You design
![navab](https://github.com/MozamelJawad/Capstone-Project/assets/6615394/a51893c9-fb80-4ebb-bbb2-5a88d672fcea)

#### Original
![navab](https://github.com/MozamelJawad/Capstone-Project/assets/6615394/b96cfa71-9777-4e5f-b1f3-416c663a77f4)